docs/exchange-operators/rosetta/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Rosetta API
+hide_title: true
+description: Learn how to install Mina’s implementation of Rosetta API.
+keywords:
+  - historical data
+  - blockchain integration
+  - explorer
+  - wallet
+  - archive node
+  - graphQL
+  - rosetta
+---
+
+# Rosetta API
+
+import Subhead from "@site/src/components/common/Subhead";
+
+<Subhead>
+Learn how to use Mina’s implementation of the Rosetta API.
+</Subhead>
+
+[Rosetta API](https://docs.cloud.coinbase.com/rosetta/docs/welcome) is an open-source specification and set of tools that make deploying and interacting with the blockchain quickly and easily.
+
+Use Rosetta API if you are:
+- Querying historical data from the Mina blockchain
+- Integrating the Mina blockchain with your exchange
+- Building blockchain applications, such as explorers and wallets
```

Comment:
I would say that using Rosetta is preferable in any case that Rosetta can cover. It's API is more appropriate for applications and improtantly, far more stable than GraphQL. The latter should only be used if there's no other choice. Sadly Rosetta does not cover mina-specific features like ZK apps.

Reply:
added a sentence that Rosetta is preferred